Gemfile

```diff
@@ -59,10 +59,15 @@ gem 'sentry-ruby'
 # Manage seeds
 gem 'dibber'
 
+# Functional UI automation
+gem 'capybara'
```

Comment:
@adeoke if you put these here you are installing them into the production app whereas I they need to be available for your tests.

Reply:
np. Ill move to the ui group

Reply:
No need because of `gem 'capybara-selenium'`

Reply:
removed